en/docs/install-and-setup/setup/reference/default-product-ports.md

```diff
@@ -12,14 +12,144 @@ This page describes the default ports used by each runtime of WSO2 API Manager.
 
 Listed below are the ports used by the API-M runtime when the [port offset]({{base_path}}/install-and-setup/setup/deployment-best-practices/changing-the-default-ports-with-offset/#configuring-the-port-offset) is 0.
 
--   5672 - Used by the internal Message Broker.
--   7611 - TCP port to receive throttling events. Required when Thrift data publisher is used for throttling.
--   7711 - SSL port for secure transport to receive throttling events. Required when Thrift data publisher is used for throttling.
--   8280 - Passthrough or NIO HTTP transport
--   8243 - Passthrough or NIO HTTPS transport
--   9611 - TCP port to receive throttling events. Required when Binary data publisher is used for throttling.
--   9711 - SSL port for secure transport to receive throttling events. Required when Binary data publisher is used for throttling.
--   9099 - Web Socket ports.
+<table>
+	<tr>
+		<th>
+			Default Port
+		</th>
+		<th>
+			Description
+		</th>
+	</tr>
+	<tr>
+		<td>
+			<code>9443</code>
+		</td>
+		<td>
+			Port of the HTTPS servlet transport. The default HTTPS URL of the management console is <code>https://localhost:9443/carbon</code>.
+		</td>
+	</tr>
+	<tr>
+		<td>
+			<code>9763</code>
+		</td>
+		<td>
+			Port of the HTTP servlet transport. The default HTTP URL of the management console is <code>http://localhost:9763/carbon</code>.
+		</td>
+	</tr>
+	<tr>
+		<td>
+			<code>10389</code>
+		</td>
+		<td>
+			Port of the embedded LDAP server.
+		</td>
+	</tr>
+	<tr>
+		<td>
+			<code>5672</code>
+		</td>
+		<td>
+			Port of the internal Message Broker of the API-M runtime.
+		</td>
+	</tr>
+	<tr>
+		<td>
+			<code>7611</code>
```

Comment:
The thrift port is not in use now AFAIK. @tharindu1st Please confirm.

Reply:
yes 7611 and 7711 we can remove from docs

Reply:
fixed